Added "rnpm" entry

DtCarrot opened the pull request.

KittyGiraudel commented on _glossary/RNPM.md:
```diff
@@ -0,0 +1,8 @@
+---
+title: rnpm (React Native Package Manager)
```
Instead of having this addition everywhere, what do you think of adding `rnpm` to [the list of abbreviations](https://github.com/HugoGiraudel/SJSJ/blob/gh-pages/_data/abbr.yml) so we have proper helper display without cluttering the content with secondary information? 

If you agree, we can remove `(React Native Package Manager)` everywhere, including in the README.

KittyGiraudel commented on _glossary/RNPM.md:
```diff
@@ -0,0 +1,8 @@
+
+# rnpm
+
+[rnpm (React Native Package Manager)](https://github.com/rnpm/rnpm) is built to ease React Native development by automatically linking native dependencies to your iOS/Android project.
```
Do you think it would be nice to add a link to `React` entry? 

``` markdown
ease [React Native](/_glossary/REACT.md#react-native) development
```

HugoGiraudel commented:
Awesome, thanks! Added 2 comments. :)

DtCarrot commented:
Hey! I have added rnpm under the list of abbreviations and removed the long word from everywhere else in the code. Added link to React Native as well.

HugoGiraudel commented:
Great! Thanks: http://jargon.js.org/_glossary/RNPM.md. :)
